docs/search/README.md

```diff
@@ -43,13 +37,9 @@ We can collect lots of this data \(100,000s of actions logged so far\), but it l
 
 * **Explicit data collection:** users are told that they are part of an experiment, and providing us with information is their primary motive.
 
-Some of the data we've used to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that people / researchers might be looking for, following a set of instructions to obtain data in a useful format.
-
-This data is small \(20-50 rows per person, per session, with ~10 people taking part\), but can be deeply contextualised by talking to the people who took part.
-
-We're now ready to start collecting explicit feedback through an [opt-in mechanism on wellcomecollection.org/works](https://github.com/wellcometrust/wellcomecollection.org/issues/4411), with all users given the option to rate the relevance of individual results in relation to their query.
+Some of the data we've used to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that people / researchers might be looking for, following a set of instructions to obtain data in a useful format. This data is small \(20-50 rows per person, per session, with ~10 people taking part\), but can be deeply contextualised by talking to the people who took part.
 
-The resulting data matches the format of the small-scale research described above, but can be collected at a much faster pace with less use of an analyst's time. The increased sample size and breadth of subjects evaluated should also give us more confidence in the 'goodness' of the changes we're making to our search algorithms.
+We've also collected data by diary studies, asking users to provide their real life search queries and apply a ranking to them and by having an opt-in relevance ranker, allowing those who wanted to rank search results could do so.
```

Comment:
Not sure the what the need for "opt-in relevance ranker" is here, as it won't make sense outside of Wellcome Collection without an example.

Reply:
Jenn suggested this be added as one of the many ways we've gathered user feedback. Could change text to read, "staff-only opt-in relevance ranker, allowing those with in-depth knowledge of the Collection to rank search results."